content/6_Advanced/FFT.mdx

```diff
@@ -17,17 +17,17 @@ export const problems = {
 			'convolution_mod',
 			'Easy',
 			false,
-			[],
-			''
+			['FFT'],
+			'convolution-mod'
```

Comment:
why are there separate solution files as well as sols within the module? just put them in the module

Reply:
it was like that before. i'll remove the internal sols then